docs/sources/tempo/release-notes/v2-3.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Version 2.3 release notes
+menuTitle: V2.3
+description: Release notes for Grafana Tempo 2.3
+weight: 45
+---
+
+# Version 2.3 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.3.
+
+This release gives you:
+
+* New [structural operators]({{< relref "../traceql#experimental-structural" >}}) in TraceQL:
+    * Ancestor (`<<`)
+    * Parent (`<`)
+    * Not-child (`!>`) (experimental)
+    * not parent (`!<`) (experimental)
+    * not siblings (`!~`) (experimental)
+
+* TraceQL support for [searching quoted attributes]({{< relref "../traceql#experimental-structural" >}}) (for example, `{ span."name with spaces" = "foo" }`).
+* [Dedicated attribute columns]({{< relref "../operations/dedicated_columns" >}}) for improving query performance on your most frequently queried attributes
+
+Tempo 2.3 makes [vParquet3]({{< relref "../configuration/parquet" >}}), a Parquet version designed to be more compatible with other Parquet implementations, available as a production-ready option.
+This block format is required for using structural operators and improves query performance relative to previous formats.
+
+Read the [**Tempo 2.3 blog post**](NEED-LINK) for more examples and details about these improvements.
```

Comment:
Doc-validator won't pass until the blog post link is here.

Reply:
we're in a bit of a chicken/egg situation here. normally i link to the release notes. i was hoping they'd be in place today/tomorrow morning so i could add the link to the blogpost

Reply:
We can always publish these and update the link to the blog post. We know what the release notes link will be -- it's on the front of the PR. 
```suggestion
Read the **Tempo 2.3 blog post** for more examples and details about these improvements.
```